### ISSUE TITLE
Github integration: Single repo access

### ISSUE THREAD

**Issue by matejminar** [getsentry/sentry#77925]
Arthur: 
- Was not able to find my repo as the Sentry integration was only allowed to access a single repo
- I set this up ages ago and there was no hint in the UI that this could be the reason I do not see other repos

Ogi:
- Connect with git providers takes me to [settings](https://telex-sandbox.sentry.io/settings/integrations/?category=source+code+management) where row github is not clickable, i have to click on the title. Annoying
- I click on add installation, takes me to github, i already have the sentry app so i allow access to this one repo.
- Sentry settings page says "not installed" even after a refresh
- Uninstalling sentry app from github completely and installing it again worked

Let's first check with folks in #discuss-integrations.

**Comment by sentaur-athena** [comment]
@matejminar we're cleaning up and prioritizing our backlog. I don't understand this ticket though 😅 can you please explain the bug?

**Comment by matejminar** [comment]
@obostjancic @ArthurKnaus could you please add more details here? I believe I just copy pasted this text from your dogfooding notes

**Comment by ArthurKnaus** [comment]
Not sure about Ogi's part.<br>Mine is more an improvement less a bug.<br>When setting up the GitHub integration you can limit the repository access on GitHub's side. E.g. I only allowed access to the repo I wanted to link.<br><br>A few months later I naturally forgot about this setting and wanted to link another repo in the Sentry UI. However, no other repository was visible and I had to figure out by myself that the integration did not have access to the others.

<img src="https://uploads.linear.app/3ed206af-f87a-40aa-82a0-150bf83e43e6/f1f1f181-cee3-4dd4-aa32-bf875bbbac6d/bfe00270-9e74-41fa-bd3f-3925a2c7df44?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzNlZDIwNmFmLWY4N2EtNDBhYS04MmEwLTE1MGJmODNlNDNlNi9mMWYxZjE4MS1jZWUzLTRkZDQtYWEzMi1iZjg3NWJiYmFjNmQvYmZlMDAyNzAtOWU3NC00MWZhLWJkM2YtMzkyNWEyYzdkZjQ0IiwiaWF0IjoxNzQ1OTk0MjIzLCJleHAiOjMzMzE2NTU0MjIzfQ.-e0161RXZtkHEl11vGSkndN7iE3op509_ctOOSSEcCI " alt="Screenshot 2025-04-30 at 08.21.04.png" width="296" data-linear-height="184" />

I think it would be great to have a link here to the integration settings on GitHub telling the user they need to make sure it has access to the repo they want to add.

**Comment by sentaur-athena** [comment]
I agree. We had this complaint from customers too. Many just report a bug saying my repo doesn't show up.
